### PR TITLE
webkit2.init.lisp: Add micro webkit version to *features*.

### PR DESCRIPTION
**Pull request by aadcg**

### CODE REVIEW

File: webkit2/webkit2.init.lisp [hunk 1]
```diff
@@ -26,12 +26,14 @@
 
 (defcfun "webkit_get_major_version" :int)
 (defcfun "webkit_get_minor_version" :int)
+(defcfun "webkit_get_micro_version" :int)
 
 (eval-when (:load-toplevel :execute)
   (with-standard-io-syntax
-    (let* ((version (format nil "~a.~a"
+    (let* ((version (format nil "~a.~a.~a"
```
Comment (aartaka) [inline webkit2/webkit2.init.lisp:33]: While we're at that, I'd add several features here (resembling Nyxt):
- Major version.
- Major+minor version.
- Full version.

**Review by aartaka — APPROVED** [review]

**Comment by aartaka** [comment]
CC @joachifm, can you give @aadcg commit access? He's likely to work on cl-webkit from time to time.

File: webkit2/webkit2.init.lisp (hunk 1 above)
Comment (aartaka) [reply webkit2/webkit2.init.lisp:33]: For WebKitGTK 2.40.1 we push three features:
- `WEBKIT-2` (already pushed as `WEBKIT2` instead).
- `WEBKIT-2.40` (pushed previously, not pushed in this PR).
- `WEBKIT-2.40.1` (pushed in this PR).

**Comment by joachifm** [comment]
> CC @joachifm, can you give @aadcg commit access? He's likely to work on cl-webkit from time to time.

Sure.  Invite sent.

**Review by aartaka — APPROVED** [review]

**Comment by aadcg** [comment]
@Ambrevar OK to merge?

Question: This PR introduces a change. `WEBKIT2` used to be member of `*features*`, but it is now replaced with `WEBKIT-2`. Should we tag a new release? Thanks.

**Comment by aartaka** [comment]
> Question: This PR introduces a change. `WEBKIT2` used to be member of `*features*`, but it is now replaced with `WEBKIT-2`. Should we tag a new release? Thanks.

I'd add this back hard-coded until we hit WebKit 3 :)

File: webkit2/webkit2.init.lisp [hunk 2]
```diff
@@ -26,26 +26,31 @@
...
-      (pushnew versym *features*)))
-
+          (pushnew (alexandria:make-keyword sym) *features*)))
+      (pushnew (alexandria:make-keyword (format nil "WEBKIT-~a" major))
+               *features*)
+      (pushnew (alexandria:make-keyword (format nil "WEBKIT-~a.~a" major minor))
+               *features*)
+      (pushnew (alexandria:make-keyword (format nil "WEBKIT-~a.~a.~a" major minor micro))
+               *features*)))
+  ;; To be deprecated when WebKit3 is released.
```
Comment (aadcg) [inline webkit2/webkit2.init.lisp:55]: As suggested [in this comment](https://github.com/joachifm/cl-webkit/pull/71#issuecomment-1527945081).

**Comment by aadcg** [comment]
@aartaka Ok to merge?

**Comment by aartaka** [comment]
OK to merge!

**Comment by aartaka** [comment]
Merged as b8ae1e2e379a297860e21beae801c91c790d5408 (how do we actually merge a singular commit? cherry-pick-donating it to `master`?)

**Comment by aadcg** [comment]
Thanks for merging! .

>  how do we actually merge a singular commit? cherry-pick-donating it to master?

In this case, when the contribution is coming from a fork, I usually "merge" it locally and then push to master. Since this a simple merge, cherry picking is a good strategy. In short, similarly to the way you did I believe.

**Comment by Ambrevar** [comment]
Thanks!

**Pull request merged**